config/drupal-vm.services.yml

```diff
@@ -2,6 +2,9 @@ parameters:
   monolog.channel_handlers:
     default: ['default_handler', 'error_handler']
   monolog.processors: ['process_id', 'message_placeholder', 'current_user', 'request_uri', 'ip', 'referer']
+  journal:
+    path: https://elifesciences.org
```

Comment:
URLs usually have been put into https://github.com/elifesciences/journal-cms-formula/blob/master/salt/journal-cms/config/srv-journal-config-local.settings.php rather than https://github.com/elifesciences/journal-cms-formula/blob/master/salt/journal-cms/config/srv-journal-config-local.services.yml
Can the same be done here?